#request
string text
int16 emotion
---
#response
bool result
